component/org.wso2.carbon.identity.api.otp.service.emailotp/src/main/resources/email-otp.yaml

```diff
@@ -167,6 +206,60 @@ components:
           description: The generated OTP
 
     #-----------------------------------------------------
+    # OTP Verification Request object
+    #-----------------------------------------------------
+    OTPVerificationRequest:
+      type: object
+      properties:
+        userId:
+          type: string
+          description: SCIM ID of the user
+          example: '8b1cc9c4-b671-448a-a334-5afe838a4a3b'
+        transactionId:
+          type: string
+          description: Unique identifier of the generated OTP
+        emailOtp:
+          type: string
+          description: EMAIL OTP
+
+    #-----------------------------------------------------
+    # OTP Verification Response object
+    #-----------------------------------------------------
+    OTPVerificationResponse:
+      type: object
+      required:
+        - isValid
+        - userId
+      properties:
+        isValid:
+          type: boolean
+          description: EMAIL OTP verified successfully
+        userId:
```

Comment:
why do we need to send back a userid?

Reply:
This feature was developed based on customer request. They specifically requested that the request object and the response object should be the same as the "/validate" API request and response.